help/destinations/catalog/personalization/algolia.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Algolia
+description: Use this connector to activate audiences to Algolia for personalization and use across search and recommendations. You can then use the Algolia User Profile source to import the profiles into the Real-Time CDP to build rich audiences.
+---
+
+# [!DNL Algolia] connection
+
+## Overview {#overview}
+
+>[!IMPORTANT]
+>
+>The [!DNL Algolia] destination connector and documentation page are created and maintained by the Algolia Integration Services team. For inquiries or update requests, contact them at [adobe-algolia-solutions@algolia.com](adobe-algolia-solutions@algolia.com).
+
+Use the [!DNL Algolia] destination connection to send Adobe Experience Platform audiences to Algolia to personalize search and recommendations. This requires the use of the [!DNL Algolia User Profiles] source connector to pull user profiles from Algolia to help build rich audiences.
+
+This tutorial provides steps to create an [!DNL Algolia] destination connection and dataflow using the Adobe Experience Platform user interface.
+
+![The destination catalog with the Algolia destination.](../../assets/catalog/personalization/algolia/catalog.png)
+
+## Use cases {#use-cases}
+
+To help you better understand how and when you should use the [!DNL Algolia] destination, here are sample use cases that Adobe Experience Platform customers can solve by using this destination.
+
+### Personalization consistency {#personalization-consistency}
+
+Use this destination connector to deliver a consistent personalization across your site from home page to search.  
+
+For example, as a marketer, you might want to build rich audiences in Adobe Experience Platform from multiple user data sources, including Algolia. You can use the [!DNL Algolia] connector to share the audiences for targeting strategies, leading to a boost in campaign personalization and conversion.
```

Comment:
Just to make it very clear to readers, please mention "[!DNL Algolia] destination connector" and "[!DNL Algolia] source connector" whenever applicable, to eliminate any confusion. This is especially important since you are also calling out the use of the source connector.